ctapipe/coordinates/transforms.py

```diff
@@ -0,0 +1,36 @@
+from astropy.coordinates import SkyCoord, AltAz
+from ..coordinates import CameraFrame
+import astropy.units as u
+
+
+
+def event_pos_in_camera(event, tel_id, horizon_frame=AltAz()):
+    """
+    Return the position of the source in the camera frame
+
+    Parameters
+    ----------
+    event: `ctapipe.io.containers.DataContainer`
+    tel_id: int - id of the telescope
+    horizon_frame: `astropy.coordinates.builtin_frames.altaz.AltAz`
+
+    Returns
+    -------
+    (x, y) (float, float): position in the camera
+    """
+
+    focal = event.inst.subarray.tel[tel_id].optics.equivalent_focal_length
+
+    telescope_pointing = SkyCoord(alt=event.mc.tel[tel_id].altitude_raw * u.rad,
+                                  az=event.mc.tel[tel_id].azimuth_raw * u.rad,
+                                  frame=horizon_frame)
+
```

Comment:
this is the alt/az location of the *event*  in the sky not the telescope, right? I would think you want to use the telescope pointing.  In the case of diffuse simulations, this would change for each event and is not what you want.

Reply:
This should be the pointing (it's mc.tel[tel_id]... not mc.altitdude)

Reply:
It is the telescope pointing.
Pointing should be in `event.pointing.tel[tel_id]` but this is not the case for MC and right now telescope pointing seems to be in `event.mc.tel[tel_id].altitude_raw`.
The event altitude is in `event.mc.alt`.

Reply:
let's fix the SimTelEventSource to put it in pointing then

Reply:
This is a long standing issue, see here: https://github.com/cta-observatory/ctapipe/issues/876